src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/BalanceSheetTransformer.java

```diff
@@ -8,4 +9,11 @@
     BalanceSheet getBalanceSheet(CurrentPeriodApi currentPeriod);
 
     CurrentPeriodApi getCurrentPeriod(BalanceSheet balanceSheet);
+
+    /**
+     * Gets the previous period data on the balance sheet
+     * @param balanceSheet populated balance sheet
```

Comment:
For consistency with our other Javadoc comments and general convention, could you add a linebreak before the `@param..` line and end the description itself with a full-stop?

Reply:
resolved